Dockerfile

```diff
@@ -132,4 +132,4 @@ COPY --from=withpackage --chown=node:node /usr/src/app/core/public ./core/public
 # needed to set the database url correctly based on PGHOST variables
 COPY --from=withpackage --chown=node:node /usr/src/app/core/.env.docker ./core/.env
 
-CMD node core/server.js
+CMD ["node", "core/server.js"]
```

Comment:
Apparently using non-json arguments here can prevent the server from responding to signals sent to the container